docs/setup/installation.md

```diff
@@ -0,0 +1,33 @@
+---
+description: Get started by installing the GitLab plugin from the Marketplace
+---
+
+# Installation
+
+### Release Dependency
+
+* The GItLab plugin is included in the Plugin Marketplace in Mattermost v5.16 and above
+* For Mattermost v5.13 and earlier a manual install is necessary
+
+### Marketplace Installation
+
+1. Go to **Main Menu &gt; Plugin Marketplace** in Mattermost
+2. Search for "gitlab" or manually find the plugin from the list and click **Install**
+3. After the plugin has downloaded and been installed, click the **Configure** button
+
+### Manual Installation
+
+If your server doesn't have access to the internet, you can download the latest [plugin binary release](https://github.com/mattermost/mattermost-plugin-gitlab/releases) and upload it to your server via **System Console > Plugins &gt; Plugin Management.**  The releases on this page are the same used by the Marketplace. 
+
+See [documentation](https://docs.mattermost.com/administration/plugins.html#set-up-guide) for more details.
+
+### Compatibility
+
+| Mattermost-Plugin-Gitlab | Mattermost | GitLab |
```

Comment:
Now the compatibility version is directly in release page https://github.com/mattermost/mattermost-plugin-gitlab/releases